src/components/AddUser.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div id="add-user">
+    <input v-model="selectedUsername" type="text" placeholder="Username" list="user-list">
+    <datalist id="user-list">
+      <option v-for="user in users" :key="user.id" :value="user.username"/>
+    </datalist>
+    <button @click="addUser">Add to board</button>
+
+    <p>Current users:</p>
+    <p v-for="user in boardUsers" :key="user.id">{{ user.username }}</p>
+  </div>
+</template>
+
+<script>
+import axios from '../axiosInstance';
```

Comment:
waarom ../ terwijl we overal @/ gebruiken?

Reply:
Die was autogenerated